.ci/setup_env_sles.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+cidir=$(dirname "$0")
+source "/etc/os-release" || source "/usr/lib/os-release"
+source "${cidir}/lib.sh"
+
+echo "Add repo for perl-IPC-Run"
+perl_repo="https://download.opensuse.org/repositories/devel:languages:perl/SLE_12_SP3/devel:languages:perl.repo"
```

Comment:
Ouch - service pack versions encoded in the URL? If there really is no alternative to this, please create variables for `12` and `SP3` and use those throughout this file as this will clearly break at some point so we need to make it quick and easy to update to v99 / sp99, etc.

Reply:
Actually the `SPn` suffix for SLE does not identify strictly a service pack. 
It is better to consider it as a new OS release, since it has its own repositories and packages set.